app/views/templates/_form.html.erb

```diff
@@ -30,14 +30,19 @@
       <%= render "custom_tabs", :f => f if type == 'ptable' %>
 
       <% if @template.locked? -%>
+        <div id="alert-warning">
```

Comment:
I added those divs because it makes easier for plugins to make changes on the alerts(defacing).